_data/navigation.yml

```diff
@@ -448,3 +441,11 @@ community:
     children:
       - title: FAQs
         url: /community/faqs/
+
+publications:
+  - title: eBook
```

Comment:
[minor] Curious what the most common spelling here is, given https://en.wikipedia.org/wiki/E-book